doc/radosgw/frontends.rst

```diff
@@ -0,0 +1,97 @@
+==============
+HTTP Frontends
+==============
+
+.. contents::
+
+The Ceph Object Gateway supports two embedded HTTP frontend libraries
+that can be configured with ``rgw_frontends``.
+
+Beast
+=====
+
+.. versionadded:: Mimic
```

Comment:
@cbodley Is this the line you said should read "versionadded:: Luminous" ?

Reply:
good catch, thanks! i remember making the change, but i must have forgotten to add it